README.md

```diff
@@ -1,24 +1,178 @@
-# README
+<a name="readme-top"></a>
 
```

Comment:
- [ ] To make your README more professional, I could be better to fix the broken link of your logo

![Screenshot 2023-10-28 010836](https://github.com/henask12/Budget-App/assets/57408419/ce142608-3324-4b75-84e5-3f2a8e17448e)